packages/hds-js/package.json

```diff
@@ -9,7 +9,7 @@
   "esnext": "./index.js",
   "types": "./index.d.ts",
   "sideEffects": [
-    "base.min-*"
+    "base.*"
```

Comment:
Now that we know this is for css and "hds-js" has no css, this sideEffects can be removed.